src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,61 @@
+package calculator;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringCalculatorTest {
```

Comment:
```suggestion
class StringCalculatorTest {
```
junit5부터는 테스트 클래스에 public 접근제어자가 제거되어도 됩니다 :)

https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods